Automattic/woocommerce-payments
# Add Download CSV button to Disputes

Fixes #3385 
This PR is heavily inspired by #2778.

#### Changes proposed in this Pull Request
Adds a Download button to the Disputes table to export the content of the table in CSV format. We  already offer this feature in other tables like Transactions and Deposits, this PR adds the same functionality to the Disputes table.

This is how the new Download button looks like in the Disputes table:
<img width="1551" alt="Screen Shot 2021-11-23 at 20 13 03" src="https://user-images.githubusercontent.com/1553182/143088937-7178dfe5-b38c-4536-90a6-09d4c23050da.png">









#### Testing instructions





* To see the Download button, you need to create some Disputes upfront. In testing mode, you can create an order in you local environment and pay with a test card (more details in [Test your integration](https://stripe.com/docs/testing#disputes) docs in Stripe) 

<img width="830" alt="Screen Shot 2021-11-23 at 20 23 29" src="https://user-images.githubusercontent.com/1553182/143090302-8d0b8760-8563-4674-8ddd-bc20d2a7f6a3.png">

_There is also the option of creating a payment from the Stripe dasboard using a test card as payment method._

* Once you have one or more disputes available you will see the Download button. Clicking on it, you will get a `csv` file.
-------------------


**Post merge**
